src/test/java/stepsDefinitions/Steps.java

```diff
@@ -1,15 +1,17 @@
-package stepsdefinition;
+package stepsDefinitions;
```

Comment:
The package should be named in lower case as convention:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html